dashboard/utils.py

```diff
@@ -69,231 +59,212 @@ def __init__(self, user, program, edx_user_data):
                     Q(edx_course_key__isnull=True) | Q(edx_course_key__exact='')
                 ).values_list("edx_course_key", "course__id")
             )
-            self.course_ids = self.edx_key_course_map.keys()
-            self.pearson_exam_status = self.get_pearson_exam_status()
+            self.edx_course_keys = set(self.edx_key_course_map.keys())
 
             if self.financial_aid_available:
-                self.paid_course_ids = set(Line.objects.filter(
-                    Q(order__status='fulfilled') & Q(course_key__in=self.course_ids) & Q(order__user=user)
+                self.paid_course_keys = set(Line.objects.filter(
+                    order__status='fulfilled', course_key__in=self.edx_course_keys, order__user=user
```

Comment:
While you're touching this, could you change `'fulfilled'` to `Order.FULFILLED`?